atomics/include/desul/atomics/Lock_Array_Cuda.hpp

```diff
@@ -9,14 +9,15 @@ SPDX-License-Identifier: (BSD-3-Clause)
 #ifndef DESUL_ATOMICS_LOCK_ARRAY_CUDA_HPP_
 #define DESUL_ATOMICS_LOCK_ARRAY_CUDA_HPP_
 
+#include <cstdint>
+
 #include "desul/atomics/Common.hpp"
 #include "desul/atomics/Macros.hpp"
 
-#ifdef DESUL_HAVE_CUDA_ATOMICS
+namespace desul {
 
-#include <cstdint>
+#ifdef DESUL_HAVE_CUDA_ATOMICS
 
-namespace desul {
```

Comment:
This is just noise.  Please avoid doing things like that.

Reply:
`clang-format` moved the header for me and we need `ensure_cuda_lock_arrays_on_device` to be available in `namespace desul` even when `CUDA` atomics are not defined since that is used in the tests unconditionally. Of course, you could also put guards in the unit tests instead. In `Kokkos` we are using that function anyway only when compiling with CUDA support.
Anyway, I updated the pull request to minimize changes related to this now.